quarkus/defaults/build.gradle.kts

```diff
@@ -17,6 +17,21 @@
  * under the License.
  */
 
-plugins { id("polaris-server") }
+plugins {
+  alias(libs.plugins.quarkus)
+}
+
+dependencies {
+  compileOnly(platform(libs.quarkus.bom))
```

Comment:
nit: maybe add a comment about IDE feature here (the PR's description is too far away when working with code) :)

Does autocompletion work for Poalris-specific properties?

Reply:
> Does autocompletion work for Poalris-specific properties?

Not very well... most of the config classes are in `polaris-service-common` or `polaris-quarkus-service`, so there is a chicken and egg problem here :-(

Reply:
Do you think it might be preferable to move `application.properties` to `quarkus/server` with some copy-paste in `quarkus/admin`?

I'm ok with this PR, but I'm not sure it's the best solution, TBH :)

Reply:
When I make this module, I was mainly trying to avoid copy-paste of config, but if it helps with IDE workflows, I suppose we can tolerate some config duplication in favour of nicer dev UX.

Also, as the admin tool matures, there's probably going to be less config overlap with the main server. WDYT?

Reply:
I think at this point, I would be in favor of leaving things where they are. I just want to get autocompletion working :-)

Reply:
SGTM :+1: